test/app.test.js

```diff
@@ -30,7 +30,7 @@ describe('app', function() {
       return redirects.indexOf(route) === -1;
     }).map(function(url) {
       if (url === '/') return '/';
-      return '/' + url;
+      return '/' + url + '/';
```

Comment:
That makes sense: app.js does a redirect; so the question is do we test to make sure that redirect does the right thin in addition to changing this string?